gio/src/list_store.rs

```diff
@@ -52,6 +64,28 @@ impl<O: IsA<ListStore>> ListStoreExtManual for O {
             )
         }
     }
+
+    fn find_with_equal_func<P: IsA<glib::Object>, F: Fn(&Object, &Object) -> glib::ffi::gboolean>(
+        &self,
+        item: &P,
+        equal_func: F,
+    ) -> Option<u32> {
+        unsafe {
+            let mut position = mem::MaybeUninit::uninit();
+            let ret = from_glib(ffi::g_list_store_find_with_equal_func(
+                self.as_ref().to_glib_none().0,
+                item.as_ref().to_glib_none().0,
+                Some(equal_func_trampoline),
+                position.as_mut_ptr(),
+            ));
+            let position = position.assume_init();
+            if ret {
+                Some(position)
```

Comment:
```suggestion
            if ret {
                let position = position.assume_init();
                Some(position)
```

Reply:
Just `Some(position.assume_init())`  should be fine :-)